app/browser/reducers/windowsReducer.js

```diff
@@ -24,6 +24,7 @@ const BrowserWindow = electron.BrowserWindow
 const firstDefinedValue = require('../../../js/lib/functional').firstDefinedValue
 const settings = require('../../../js/constants/settings')
 const getSetting = require('../../../js/settings').getSetting
+const { shouldDebugStoreActions, shouldDebugTabEvents } = require('../../cmdLine')
```

Comment:
Thanks for the note @jumde! I haven't satisfied linter and some tests yet. Some of this code was written in an experimental mode with things added, removed, or changed very quickly. But right now we're pretty locked down on architecture and are in clean-up mode, so that's what I'm looking in to over the course of the next day or 2.